components/Form/AddressLookup/AddressLookup.spec.tsx

```diff
@@ -30,6 +34,16 @@ describe('AddressLookup', () => {
         'You must enter a valid postcode'
       );
       expect(validate.postcode({ postcode: 'e83as' })).toBe(true);
+      expect(validate.buildingNumber()).toBe(
+        'Building number must use valid characters (0-9)'
+      );
+      expect(validate.buildingNumber({ buildingNumber: '123' })).toBe(true);
+      expect(validate.buildingNumber({ buildingNumber: '123a' })).toBe(
+        'Building number must use valid characters (0-9)'
+      );
+      expect(validate.buildingNumber({ buildingNumber: '' })).toBe(
+        'Building number must use valid characters (0-9)'
+      );
```

Comment:
I think these tests warrant a refactoring as the test descriptions don't document the function very well and each test tests different behaviours but for this PR maybe we could separate out the building number related behaviours?

Reply:
Agreed, have split them out into their own describe blocks and made them more descriptive